rosdep/base.yaml

```diff
@@ -3463,6 +3463,12 @@ pocketsphinx:
     xenial: [pocketsphinx]
     yakkety: [pocketsphinx]
     zesty: [pocketsphinx]
+polyclipping-dev:
+  debian:
+    jessie: [libpolyclipping-dev]
+    stretch: [libpolyclipping-dev]
+    sid: [libpolyclipping-dev]
```

Comment:
We usually don't target rolling distros so you don't have to add sid to this list (same below)